### PR TITLE
Enable the Koji repository in the RPM tests

### PR DESCRIPTION
**Pull request by poncovka**
Use the latest updates to fix our issues with the RPM tests.

### CODE REVIEW

**Comment by poncovka** [comment]
I have no idea what is actually broken, but enabling the Koji repository seems to resolve the issues with RPM tests.

File: Makefile.am [hunk 1]
```diff
@@ -75,7 +75,7 @@ TEST_INST_BUILD_DIR = $(BUILD_RESULT_DIR)/02-test-install
 
 CONTAINER_ENGINE ?= podman
 # build/run tweaks for all containers, such as --cap-add
-CONTAINER_BUILD_ARGS ?= --no-cache
+CONTAINER_BUILD_ARGS ?= --no-cache --pull-always
```
Comment (jkonecny12) [inline Makefile.am:78]: Already part of https://github.com/rhinstaller/anaconda/pull/3812.

**Review by jkonecny12 — CHANGES_REQUESTED** [review]
This solution is great for test that there is already a fix in koji but I would like to avoid having Koji as our repository. It could break any second and there is no testing of that.

**Comment by jkonecny12** [comment]
Seems that this issue is now resolved even without this PR. Tested locally and I'll also run the container refresh to be sure.

**Comment by jkonecny12** [comment]
Tested, we have working infra again - we can close this now.

**Comment by poncovka** [comment]
Well, I am going to fix the next breakage the same way. We can discuss this solution at Monday's meeting.

**Pull request closed without merging**